src/Commands/Sql/ISqlServerLister.cs

```diff
@@ -0,0 +1,6 @@
+namespace AzureMcp.Commands.Sql;
```

Comment:
Please put the interface into src/Services/Interfaces/I{Service}Service.cs

Reply:
You can get more details in https://github.com/Azure/azure-mcp/blob/main/src/Docs/new-command.md